### DISCUSSION TITLE
Appreciation

### DISCUSSION THREAD

**Discussion by m-soltani** [Azure/Azure-Verified-Modules#1050] [General]
Dear Team,

I wanted to take a moment to express my appreciation for your hard work. Thanks to your efforts, we successfully migrated all our in-house developed templates and modules to use Azure verified modules, significantly improving our productivity.

Thank you for your dedication and excellent work. I hope, at some point, I am also able to contribute to this work.

**Comment by segraef** [comment]
Thank you very much @m-soltani, this, for sure, means a lot to the team!
